python/tests/test_tree_stats.py

```diff
@@ -1768,6 +1768,205 @@ class TestSiteDivergence(TestDivergence, MutatedTopologyExamplesMixin):
     mode = "site"
 
 
+############################################
+# Genetic relatedness
+############################################
+
+
+def site_genetic_relatedness(
+    ts, sample_sets, indexes, windows=None, span_normalise=True, proportion=True
+):
+    out = np.zeros((len(windows) - 1, len(indexes)))
+    samples = ts.samples()
+    haps = ts.genotype_matrix(isolated_as_missing=False).T
+    haps_mean = haps.mean(axis=0)
+    haps_centered = haps - haps_mean
+    denom = 1
+    if proportion:
+        all_samples = list({u for s in sample_sets for u in s})
+        denom = ts.segregating_sites(
+            sample_sets=all_samples, windows=windows, mode="site"
+        )
+    for j in range(len(windows) - 1):
+        begin = windows[j]
+        end = windows[j + 1]
+        site_positions = [x.position for x in ts.sites()]
+        for i, (ix, iy) in enumerate(indexes):
+            X = sample_sets[ix]
+            Y = sample_sets[iy]
+            # site_in_window = False
```

Comment:
I'm not sure where the ``site_in_window`` thing came from - was this copied from another similar function? If so, let's delete the commented out stuff here.

Reply:
Yep, I had copied over from divergence, will delete!

Reply:
heh, whoops - I should do a cleanup, i guess

Reply:
Oh no, you actually use `site_in_window` in divergence! I just commented it out for genetic_relatedness.